src/ir/block_builder_test.cc

```diff
@@ -190,10 +191,13 @@ TEST_F(BlockBuilderTest, AddImplementationMakingMultipleUpdates) {
   const NamedValueMap& results = block->results();
   SsaNames ssa_names;
   ASSERT_EQ(results.count("entity_output"), 1);
-  EXPECT_EQ(results.at("entity_output").ToString(ssa_names), "%0.entity_value");
+  //%0 is the entity_value OperationResult Value :
```

Comment:
It is not guaranteed that  `Value(value::OperationResult(op, "entity_value"))` will always be `%0`. There is a similar concern with the other value. 

Why don't you populate `ssa_names` so thaValue(value::OperationResult(op, "entity_value"))` maps to `"entity_value"` and Value(value::OperationResult(op, "primitive_value"))` maps to  
 `"primitive_value"`. 
 
Then the test will not be fragile.

Reply:
Moving Value changes and discussions to another PR.